### PR TITLE
Move `Identify Cluster` to spec

### PR DESCRIPTION
**Pull request by andy31415**
Moved to revision 4 according to the 1.2 spec:

- added new Query feature
- Added IdentifyQuery and IdentifyQueryResponse

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml [hunk 1]
```diff
@@ -62,6 +62,18 @@ limitations under the License.
       <arg name="IdentifyTime" type="int16u"/>
       <access op="invoke" privilege="manage"/>
     </command>
+    <command source="client" code="0x01" name="IdentifyQuery" optional="true" response="IdentifyQueryResponse">
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml:65]: This is not part of Matter.  See device library section "1.1.7.2. Element Requirements" for the Base Device type.  The QRY feature is never allowed to be implemented for Matter, and we should not have any of this stuff in the XML here.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml (hunk 1 above)
Comment (andy31415) [reply src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml:65]: ok, I was trusting the scraper in this case.

Maybe these requirements are not clearly enough zigbee (and my personal opinion, in a *matter* specification we should not have stuff that is *not matter/disallowed in matter*).

**Comment by andy31415** [comment]
Closing. created https://github.com/csa-data-model/projects/issues/384 for the scraper.

File: src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml (hunk 1 above)
Comment (andy31415) [reply src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml:65]: I was looking at the 1.2 app cluster specifications , where 1.2 is the Identify Cluster and I cannot find anything about zigbee. This is very very bad for spec readers. IThe spec says:

```
1.2.4.1. Query Feature
This feature supports a unicast, groupcast or multicast query of the cluster state, with a response
back to query initiator, if the identification state is active.
This feature is supported for underlying stacks that support a response to a multicast or groupcast
command.
```

I guess the "stacks that support a response to a multicast or groupcast command" is a hint there, but why do we even have it. If matter does not support this response, why do we have this in a MATTER spec? At least it should have a Zigbee only somewhere or better, `reserved bit to not overlap with Zigbee` or something like that.

Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml:65]: I don't disagree with any of the above.  The point is: you can't tell which parts of this cluster are Matter or not without looking at the device library spec...

**Pull request closed without merging**